RELEASE_NOTES.md

```diff
@@ -1,5 +1,23 @@
 # Release Notes
 
+## 2.1.19 "TBD" - TBD
+
+* This release adds support for capturing Kubernetes container names to the kubernetes monitor via log line attributes.
+  You can configure a container name as a Kubernetes log attribute using the ``${k8s_container_name}`` config template 
```

Comment:
You can include the container name as a log line attribute using the ``${k8s_container_name}`` config template syntax: